src/page.ts

```diff
@@ -1,6 +1,10 @@
 import { deadline } from "https://deno.land/std@0.201.0/async/deadline.ts";
 
-import { Celestial, Network_Cookie } from "../bindings/celestial.ts";
+import {
+  Celestial,
+  Network_Cookie,
+  Network_ResourceType,
+} from "../bindings/celestial.ts";
```

Comment:
Maybe these should be imported as types instead of really being imported? I think it was a mistake of mine to not do that in the first place.
```suggestion
import type {
  Celestial,
  Network_Cookie,
  Network_ResourceType,
} from "../bindings/celestial.ts";
```